test/util/mock-imported-components.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Return true if `value` "looks like" a React/Preact component.
+ */
+function isComponent(value) {
+  return (
+    typeof value === 'function' &&
+    Object.prototype.hasOwnProperty.call(value, 'propTypes') &&
```

Comment:
Note: Small change on this line from the client to pass lint.

Reply:
We'll need to modify this if/when we remove `propTypes` from the components in this package. See the current version of this module in the client repo.